Tools/CISupport/Shared/de-duper

```diff
@@ -96,16 +97,20 @@ def main(args):
             sha_value, extension = existing_file.name.split('.', 1)
             if extension != 'png':
                 continue
-            if not existing_file.inode() <= 1:
+            connected_to = existing_file.inode()
+            if connected_to <= 1:
                 if args.dry_run:
                     print(f'[dry-run] Removing {existing_file.path}')
                 else:
                     os.remove(existing_file.path)
                 cleaned_up += 1
             else:
-                existing_links.add(sha_value)
+                existing_links[sha_value] = connected_to
 
     for root, _, files in os.walk(args.root, topdown=False):
+        if root == args.storage:
+            continue
```

Comment:
This seems like a noteworthy change, but also we probably want `os.path.samefile` here, rather than relying on the paths being identical.